tests/_executor_mixin.py

```diff
@@ -57,6 +57,7 @@ def _running_children_with_cmdline(p):
     all_children = _direct_children_with_cmdline(p)
     workers = [(c, cmdline) for c, cmdline in all_children
                if (u'semaphore_tracker' not in cmdline and
```

Comment:
You should also remove the `semaphore_tracker` as it is not here anymore. No?

Reply:
I was keeping this line in case a `multiprocessing.sempahore_tracker` was launched for some reason during a test.